src/todos/todos.service.ts

```diff
@@ -22,4 +22,9 @@ export class TodosService {
     }
     return foundTodo;
   }
+
+  async deleteTodoById(id: string): Promise<void> {
+    const todoItem = await this.getTodoById(id);
+    await this.todoRepository.getEntityManager().removeAndFlush(todoItem);
```

Comment:
```suggestion
    await this.todosRepository.getEntityManager().removeAndFlush(todoItem);
```
[NIT] - I think it is imported in plural